packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -1227,6 +1221,13 @@ format is SerializationFormat.Bytes_Base64 or SerializationFormat.Bytes_Base64Ur
             };
         }
 
+        private static bool ValueTypeIsInt(Type valueType) =>
+            valueType == typeof(long) ||
+            valueType == typeof(int) ||
+            valueType == typeof(short) ||
+            valueType == typeof(sbyte) ||
+            valueType == typeof(byte);
+
         private static bool ValueTypeIsNumber(Type valueType) =>
             valueType == typeof(decimal) ||
             valueType == typeof(double) ||
```

Comment:
consider to remove the duplicates by using `ValueTypeIsInt`

Reply:
update to re-use `ValueTypeIsInt`